modules/margin/src/main/java/com/opengamma/sdk/margin/v3/InvokerMarginClient.java

```diff
@@ -216,6 +221,49 @@ public MarginCalcResult calculate(Ccp ccp, MarginCalcRequest request) {
   }
 
   @Override
+  public MultiCcpMarginCalcResult calculate(
+      List<Ccp> ccps,
+      MarginCalcRequest request) throws ExecutionException, InterruptedException {
+
+    Map<Ccp, CompletableFuture<MarginCalcResult>> completableFutures = ccps.parallelStream()
+        .collect(Collectors.toMap(Function.identity(), ccp -> calculateAsync(ccp, request)));
```

Comment:
Using `parallelStream()` is not necessary, as calling `calculateAsync()` should be quick.